style.css

```diff
@@ -0,0 +1,178 @@
+body {
+    font-family: 'Roboto', sans-serif;
+}
+
+header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 20px;
+    border-bottom: 1px solid lightgrey;
+    background-color: #ffffff;
+    box-shadow: 0 4px 10px -2px lightgray;
+}
+
+body {
+    background-color: #f8fafc;
+}
+
+.logos {
```

Comment:
probably should be singular (.logo) as the styling applies to each image independently and not to them altogether (as a container)